xla/stream_executor/device_description.h

```diff
@@ -68,6 +68,10 @@ struct CudaComputeCapability {
     return !(*this < CudaComputeCapability{other_major, other_minor});
   }
 
+  bool Is(int other_major, int other_minor = 0) const {
+    return !(*this == CudaComputeCapability{other_major, other_minor});
```

Comment:
Why the `!`? Also don't think this function is necessary. In cudnn_norm_rewriter.cc, you should just check the major version is AMPERE or HOPPER.